ntoskrnl/io/pnpmgr/devnode.c

```diff
@@ -0,0 +1,400 @@
+/*
```

Comment:
Diff of "just move all the functions to this file" -> what's actually in the file:
https://gist.github.com/ThFabba/1cedf75456857db56d94cd99c05081f8

Reply:
I indicated this in the commit message!